### PR TITLE
Create newfeature.txt

### PR DESCRIPTION
**Pull request by zachzloof**

### CODE REVIEW

File: newfeature.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+blah blah
```
Comment (theJAtkinson) [inline newfeature.txt:1]: This is a review

**Review by theJAtkinson — APPROVED** [review]
This is a very good review

**Review by zachzloof — COMMENTED** [review]
blah blah

File: newfeature.txt (hunk 1 above)
Comment (theJAtkinson) [inline newfeature.txt:1]:
```suggestion
blah blah blah
```

**Review by theJAtkinson — CHANGES_REQUESTED** [review]

**Pull request merged**